bsve/server/bsve_wms.py

```diff
@@ -103,3 +124,26 @@ def createBsveDataset(self, params, layer_type):
     createBsveSource.description = (
         Description('Create bsve datasets from bsve geoserver')
     )
+
+    def getLayers(self, user):
+        folder = findDatasetFolder(
+            user, user
+        )
+        if not folder:
+            return []
+
+        items = self.model('folder').childItems(folder)
+
+        layers = {}
+        for item in items:
+            adapter = item.get('meta', {}).get('minerva', {}).get('adapter')
+            name = item.get('meta', {}).get('minerva', {}).get('type_name')
+            if adapter == 'bsve':
+
+                # delete duplicates if they exist
+                if name in layers:
+                    self.model('item').remove(item)
```

Comment:
@jbeezley I like most of the changes.. just wondering if we should throw a warning if duplicate exists?